docs/quickstart.rst

```diff
@@ -91,7 +89,7 @@ If you go to ``${RV_QUICKSTART_OUT_DIR}`` you should see a directory structure l
         ├── log.csv
         ├── model-bundle.zip
         ├── tb-logs
-        │   └── events.out.tfevents.1585513048.086fdd4c5530.214.0
```

Comment:
I would suggest skipping this change. These numbers are not important and different for every run.